### DISCUSSION TITLE
How to interact with `eww`?

### DISCUSSION THREAD

**Discussion by TornaxO7** [elkowar/eww#573] [Q&A] [answered]
Hello! Assuming that I've started the daemon by opening a widget. Do I always have to apply `-c <path>` if I don't have my widgets in `~/.config/eww`?

**Comment by viandoxdev** [comment, marked as answer]
Yes, if you want that to be shorter you should symlink the config dir to ~/.config/eww (provided you don't have a config there already) or use an alias in the calling shell. In practice eww is most often invoked by script so the added length of -c ... shouldn't be a problem.

Edit: to clarify each different config path will result in an entirely separate daemon running, this is why you need to specify -c on every invocation because otherwise eww wouldn't know which daemon to communicate with.

**Reply by TornaxO7** [reply]
Oh ok.
